docs/deprecations.md

```diff
@@ -39,3 +39,9 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 ## `bin/serverless`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="DEPRECATED_ALIAS_SLSS"><div>&nbsp;</div></a>
```

Comment:
Let's add new deprecation as _first_ (see https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md#configuring-deprecations).

Ideally newest are at the top, oldest (most likely least relevant) at the bottom

Reply:
Sorry, I didn't know (and now that you said it, it makes a lot of sense).